>>>     $4^(-2)

thread '<unnamed>' panicked at src/sym/mod.rs:596:40:
Exponent too large.: TryFromBigIntError { original: () }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

thread 'main' panicked at src/main.rs:57:26:
called `Result::unwrap()` on an `Err` value: Any { .. }
